src/coreclr/inc/readytorun.h

```diff
@@ -39,6 +39,7 @@
 // R2R Version 10.0 adds support for the statics being allocated on a per type basis instead of on a per module basis, disable support for LogMethodEnter helper
 // R2R Version 10.1 adds Unbox_TypeTest helper
 // R2R Version 11 uses GCInfo v4, which encodes safe points without -1 offset and does not track return kinds in GCInfo
+// R2R Version 12 requires all return buffers to be always on the stack
```

Comment:
https://github.com/dotnet/runtime/blob/main/docs/design/coreclr/botr/clr-abi.md#return-buffers should be updated too